proxy/stat/api.py

```diff
@@ -2,11 +2,24 @@
 
 from typing_extensions import TypedDict
 
-from common.utils.pydantic import BaseModel
+from common.ethereum.hash import EthAddressField
+from common.solana.pubkey import SolPubKeyField
+from common.utils.pydantic import BaseModel, HexUIntField
 
 STATISTIC_ENDPOINT = "/api/v1/statistic/"
 
 
+class OpResourceNeonBalanceData(BaseModel):
```

Comment:
See my comment about the name of the metric.
IMHO it is better to rename the structure to something like:
`OpResourceEarnedBalanceData`

Reply:
This is done too.